agave_app/SerializeV1.h

```diff
@@ -99,6 +99,15 @@ struct LutParams_V1
   }
   NLOHMANN_DEFINE_TYPE_INTRUSIVE(LutParams_V1, window, level, isovalue, isorange, pctLow, pctHigh, controlPoints, mode)
 };
+
+struct ColorMap
+{
+  std::string name = "none";
+  std::vector<ControlPointSettings_V1> stops;
+  bool operator==(const ColorMap& other) const { return name == other.name && stops == other.stops; }
+  NLOHMANN_DEFINE_TYPE_INTRUSIVE_WITH_DEFAULT(ColorMap, name, stops)
```

Comment:
this struct is here in SerializeV1.h because it is used exclusively for saving/loading from file as json.  The data structures for file i/o are separated from the internal structures of the app.  Even if they tend to be the same, sometimes there are reasons why they can't be.